### DISCUSSION TITLE
How to resolve invalid email addresses？

### DISCUSSION THREAD

**Discussion by AhYi8** [FreshRSS/FreshRSS#6560] [Q&A]
I encountered the following error while checking the validity of my RSS feed. How can I resolve it?
![Invalid email address](https://github.com/FreshRSS/FreshRSS/assets/56581248/f5a22ca9-3a24-422f-8387-979b932207d0)

**Comment by Alkarex** [comment]
Well, the validator gives you the answers: An e-mail should be with an e-mail format, and your URL references should have a scheme such as `https://...`
That being said, those specific errors should not prevent your feed from working in FreshRSS.
